tests/moses/main/moses-framework.h

```diff
@@ -222,6 +222,32 @@ void moses_test_combo(vector<string> arguments,
     if (f_it != expected_tr_strs.end())
          unlink(cmd_tmp.second.c_str());
 }
+
+// test the atomese program candidates
+void moses_test_atomese(vector<string> arguments,
+						vector<string> expected_atomese_strs)
```

Comment:
Should be aligned with spaces rather than tabs (and use const ref instead of copy).